include/deal.II/fe/fe_p1nc.h

```diff
@@ -0,0 +1,170 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 - 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__fe_p1nc_h
+#define dealii__fe_p1nc_h
+
+#include <deal.II/base/config.h>
+#include <deal.II/fe/fe.h>
+#include <deal.II/base/quadrature.h>
+#include <deal.II/base/qprojector.h>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+
+/*!@addtogroup fe */
+/*@{*/
+
+
+class FE_P1NC : public FiniteElement<2,2>
```

Comment:
could you please add some doxygen description of the class? see https://www.dealii.org/developer/doxygen/deal.II/classFE__Q.html for an example.

Of corse users could google for 

> P1 nonconforming element

but it is still good to have a (short) description here perhaps with some plots so that there is no ambiguity in what is implemented.